README.md

```diff
@@ -11,16 +11,14 @@
 
 
 [![Licence](https://img.shields.io/github/license/midlevel/mlapi.svg?color=informational)](https://github.com/MidLevel/MLAPI/blob/master/LICENSE)
-[![Website](https://img.shields.io/badge/docs-website-informational.svg)](https://mlapi.network/)
-[![Wiki](https://img.shields.io/badge/docs-wiki-informational.svg)](https://mlapi.network/wiki/home/)
-[![Api](https://img.shields.io/badge/docs-api-informational.svg)](https://mlapi.network/api/home/)
+[![Website](https://img.shields.io/badge/docs-website-informational.svg)](https://docs-multiplayer.unity3d.com/)
+[![Api](https://img.shields.io/badge/docs-api-informational.svg)](https://docs-multiplayer.unity3d.com/docs/mlapi-api/introduction)
 
 
 The Unity MLAPI (Mid level API) is a framework that simplifies building networked games in Unity. It offers **low level** access to core networking while at the same time providing **high level** abstractions. The MLAPI aims to remove the repetitive tasks and reduces the network code dramatically, no matter how many of the **modular** features you use.
 
 ### Getting Started
-To get started, check the [Wiki](https://mlapi.network/wiki/).
-This is also where most documentation lies. Follow the [quickstart](https://mlapi.network/wiki/installation/), join our [Discord](http://discord.mlapi.network/) and get started today!
+To get started, check the [Multiplayer Docs Site](https://docs-multiplayer.unity3d.com/).
```

Comment:
Should we link to the get started guide page specifically here? Or like we did before do both. 1 link for general docs + 1 for getting started.

Reply:
It's not live on the prod site yet - I am just adding the basics for now.